### ISSUE TITLE
Cannot launch WSL

### ISSUE THREAD

**Issue by IvoVieira** [microsoft/WSL#9886]
### Windows Version

Microsoft Windows [Version 10.0.19044.2728]

### WSL Version

WSL version: 1.1.6.0

### Are you using WSL 1 or WSL 2?

- [ ] WSL 2
- [X] WSL 1

### Repro Steps

I type `wsl` on PowerShell

### Expected Behavior

To launch WSL / Ubuntu

### Actual Behavior

Ubuntu doesn't launch and I get the output:

Processing fstab with mount -a failed.
Failed to mount C:\, see dmesg for more details.
Failed to mount D:\, see dmesg for more details.

<3>WSL (7) ERROR: CreateProcessEntryCommon:368: getpwuid(1000) failed 2
<3>WSL (7) ERROR: CreateProcessEntryCommon:372: getpwuid(0) failed 2
<3>WSL (7) ERROR: CreateProcessEntryCommon:574: execvpe /bin/sh failed 2
<3>WSL (7) ERROR: CreateProcessEntryCommon:583: Create process not expected to return

**Comment by Franky1** [comment]
I had the same issue since a week and also **Docker Desktop** stopped working.

What do you get with `wsl --list` command?
I found out that i got this:

```log
docker-desktop-data (Standard)
docker-desktop
```

I set the default distro with
```powershell
wsl --setdefault docker-desktop
```

After this at least `wsl` worked again.
However **Docker Desktop** still did not start.
I uninstalled it and installed the latest version of Docker Desktop, than it worked again.

**Comment by IvoVieira** [comment]
Thank you for trying to help.

With the `wsl --list` I got:

`Windows Subsystem for Linux Distributions:
Ubuntu (Default)`

Ubuntu is the distro I'd like to use, so I think it's everything OK with it.
I've also uninstalled and re-installed the Ubuntu app from MS Store without any difference in the outcome. I also re-installed WSL too.

But again, whenever I run `wsl` I always get the errors:

`Processing fstab with mount -a failed.
Failed to mount C:\, see dmesg for more details.
Failed to mount D:\, see dmesg for more details.

<3>WSL (7) ERROR: CreateProcessEntryCommon:368: getpwuid(1000) failed 2
<3>WSL (7) ERROR: CreateProcessEntryCommon:372: getpwuid(0) failed 2
<3>WSL (7) ERROR: CreateProcessEntryCommon:574: execvpe /bin/sh failed 2
<3>WSL (7) ERROR: CreateProcessEntryCommon:583: Create process not expected to return`
